index.md

```diff
@@ -8,6 +8,573 @@ This is some welcome text.
 
 This is some helpful description of the C0111 error.
 
+
+
+
+### R0201: No self use {#R0201}
+
+If a method (a function in a class) does not make use of the 'self' (or
+first) argument, that means the function is not performing anything that
+is related to the object itself. This means the function could be moved
+outside of the class since none of the code inside makes use of anything
+inside the class it's defined.
+
+```python
+class NoSelfUsage:
+    def __init__(self):
+        self.a = 42
+
+    def no_self(self, num):
+        num = num + 2
+        print(num)
+
+# You would fix it as follows by moving it outside the class:
+def no_self(num):
+	num = num + 2
+    print(num)
+```
+
+
+
+### E0202: Method hidden {#E0202}
+
+If you accidentally mask a method with a field, it can cause other code to
+attempt to invoke what it believes to be a method, which will fail since
+it has become a field instead (a variable). The syntax will error out, and
```

Comment:
The phrase "syntax will error out" is not good. If there's a syntax error raised during parsing, say that. Otherwise, say that an error is raised when the program is run.